app/assets/stylesheets/core/_elements.scss

```diff
@@ -6,14 +6,3 @@ html {
 ul li:before {
   display: none;
 }
-
```

Comment:
this does affect layout across app. but I clicked around a bit and didn't see anything too weird. The design standards should be defining these types of default styles for us.

The reason I removed in this PR is that the selectize lib puts a `span` around highlighted elements while you are searching. Because we have special `span` styles, the highlighted bits looked super weird when we had these styles defined. Example:

Before:

<img width="402" alt="screen shot 2016-07-08 at 9 13 26 am" src="https://cloud.githubusercontent.com/assets/601515/16693907/4b85df40-44ec-11e6-9571-a2a06a82a982.png">


After:

<img width="378" alt="screen shot 2016-07-08 at 9 13 39 am" src="https://cloud.githubusercontent.com/assets/601515/16693910/4f7a889e-44ec-11e6-9bbe-4629922acf07.png">